lib/internal/test_runner/utils.js

```diff
@@ -250,45 +263,130 @@ function countCompletedTest(test, harness = test.root.harness) {
 }
 
 
-function coverageThreshold(coverage, color) {
-  coverage = NumberPrototypeToFixed(coverage, 2);
-  if (color) {
-    if (coverage > 90) return `${green}${coverage}${color}`;
-    if (coverage < 50) return `${red}${coverage}${color}`;
-  }
-  return coverage;
+function addTableLine(prefix, width) {
+  return `${prefix}${'-'.repeat(width)}\n`;
```

Comment:
```suggestion
  return `${prefix}${StringPrototypeRepeat('-', width)}\n`;
```
also consider memoizing, like https://github.com/nodejs/node/blob/102540aef9024ea6a8218a6d782243f1391612d2/lib/internal/test_runner/reporter/tap.js#L88-97